thirdeye/thirdeye-frontend/app/pods/components/filter-bar/component.js

```diff
@@ -88,26 +99,37 @@ export default Ember.Component.extend({
         isHidden
       });
     });
+
+    const eventType = filterBlocks[0].eventType;
```

Comment:
nit: can filterBlocks be `null` or `undefined` here?

Reply:
Assuming the config file is passed and not empty, filterBlocks will never be null or undefined.